{{cookiecutter.github_repository}}/settings/development.py

```diff
@@ -19,6 +19,7 @@
 TEMPLATES[0]['OPTIONS']['debug'] = DEBUG  # noqa: F405
 
 INTERNAL_IPS = ('127.0.0.1', )
+ALLOWED_HOSTS = ('*',)
```

Comment:
Let's not have this change as part of this PR. You can create a separate one, as required.